src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -328,6 +334,49 @@ protected ulong ClearRegister
             set {  *(ulong*)(_registerViewPointer->GPCLR) = value; }
         }
 
+        /// <summary>
+        /// Returns the pperipheral base address of the raspberry pi based on the ranges set within the device tree.
```

Comment:
NIT: `pperipheral` -> `peripheral`

Reply:
Changed.... Will be in the next PR Update